backend/uclapi/requirements.txt

```diff
@@ -8,7 +8,9 @@ cx-Oracle==5.3
 Django==1.11.1
 django-cors-headers==2.0.2
 django-dotenv==1.4.1
+django-redis==4.8.0
 djangorestframework==3.6.3
+fakeredis==0.8.2
```

Comment:
It is used for dev environment so that you can dev against it & your tests pass without needing a real Redis instance.